back/boxtribute_server/business_logic/box_transfer/shipment/mutations.py

```diff
@@ -17,23 +15,13 @@ def resolve_create_shipment(*_, creation_input):
         permission="shipment:create",
         base_ids=[creation_input["source_base_id"], creation_input["target_base_id"]],
     )
-    agreement = TransferAgreement.get_by_id(creation_input["transfer_agreement_id"])
-    organisation_ids = [agreement.source_organisation_id]
-    if agreement.type == TransferAgreementType.Bidirectional:
-        organisation_ids.append(agreement.target_organisation_id)
-    authorize(organisation_ids=organisation_ids)
     return create_shipment(**creation_input, user=g.user)
 
 
 @mutation.field("updateShipment")
```

Comment:
AGREED: split up into two mutations - one for sending and one for receiving

Reply:
https://trello.com/c/SEcv4ec6/1120-20-extension-of-shipment-graphql-api-for-reception-sprint-feb-20